ONSdigital/eq-author
# Checkbox answer

### What is the context of this PR?

- Wires up the "checkbox" answer type component
- Options are deletable when more than one is present
- Focus should be moved to the first "label" field when a new option is added, or a new answer

#### Bonus content

- re-named `queries` folder to `graphql`

### How to review 

- Load/create a survey
- Add a "Checkbox" answer
- Ensure focus is on "label" field
- Add another option
- Ensure focus as above
- Delete an option
